example/src/server/schema.js

```diff
@@ -50,7 +50,7 @@ const resolvers = {
       return store.todos;
     },
     todo: (root, args, context) => {
-      return store.todos.find(a => (a.id = args.id));
+      return store.todos.find(a => a.id == args.id);
```

Comment:
Nice catch! Can you change this to a triple equal though please? :)

Reply:
You bet!  I mistakenly thought you were using == in other spots.